src/tileset/wangset.rs

```diff
@@ -0,0 +1,101 @@
+use std::collections::HashMap;
+
+use xml::attribute::OwnedAttribute;
+
+use crate::{
+    error::Error,
+    properties::{parse_properties, Properties},
+    util::{get_attrs, parse_tag, XmlEventResult},
+    Result, TileId,
+};
+
+mod wang_color;
+pub use wang_color::WangColor;
+mod wang_tile;
+pub use wang_tile::{WangId, WangTile};
+
+/// Undocummented WangSet types
+#[derive(Debug, PartialEq, Clone, Copy)]
+pub enum WangType {
```

Comment:
It's technically `WangSetType`, but I see this gets a bit ugly with Rust's underscore naming style for variables. Would `wangset_type` be suitable as the variable name, then?

Reply:
Ok, let's be consistent and go with `WangSet` and `wang_set`, and thus here `WangSetType` and `wang_set_type`.